deploy-disconnected-registry/common.sh

```diff
@@ -90,6 +90,11 @@ function trust_internal_registry() {
     update-ca-trust extract
     echo ">> Done!"
     echo
+
+    # Add certificate to OpenShift configuration
+    oc --kubeconfig=${KBKNFG} create configmap ztpfwregistry -n openshift-config --from-file=ztpfw-registry-ztpfw-registry.apps.test-ci.alklabs.local:5000=${PATH_CA_CERT}
```

Comment:
Will this particular FQDN actually fix the issue we see running the pipeline?

Reply:
No, it was set to ztpfw to add the nickname, it's updated in the current PR